chapters/package-information.md

```diff
@@ -1469,7 +1469,109 @@ EXAMPLE 2 RDF: Property `spdx:purpose` in class `spdx:Package`
     <packagePurpose rdf:resource="packagePurpose_container" />
 </Package>
 ```
+  
+## 7.25 Release Date <a name="7.25"></a>
+### 7.25.1 Description
 
+This field provides a place for recording the date the package was released.
+
+Table 37 — Metadata for the release date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 7.25.2 Intent
+
+The release date is helpful for strict identification of the prerequisite assumptions of usage.
+
+### 7.25.3 Examples
+
+EXAMPLE 1 Tag: `ReleaseDate:`
+
+```text
+ReleaseDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:releaseDate` in class `spdx:Package`
+
+```text
+<Package rdf:about="...">
+    <releaseDate> 2010-01-29T18:30:22Z </releaseDate>
+</Package>
+```
+
+## 7.26 Built Date <a name="7.26"></a>
+### 7.26.1 Description
+
+This field provides a place for recording the actual date the package was built.
+
+Table 38 — Metadata for the built date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 7.26.2 Intent
+
+The date when the package was built is helpful for strict identification of the prerequisite assumptions of usage.  
+Ideally it should be recorded from build system tools directly or the creation date of the package itself.
+
+### 7.26.3 Examples
+
+EXAMPLE 1 Tag: `BuiltDate:`
+
+```text
+BuiltDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:builtDate` in class `spdx:Package`
+
+```text
+<Package rdf:about="...">
+    <builtDate> 2010-01-29T18:30:22Z </builtDate>
+</Package>
+```
+## 7.27 Valid Until Date <a name="7.27"></a>
```

Comment:
```suggestion
## 7.27 Supported Until Date <a name="7.27"></a>
```

I would prefer 'supported' to be used in the name of this field - software is 'valid' indefinitely if the users are happy to continue their use without support!

Reply:
@yoshi-i @kestewart Do you agree with @seabass-labrax proposed name change above?

Reply:
Hi @seabass-labrax,  this option was part of the earlier discussion, but "support" doesn't really cover all the use cases (although it is certainly one).   There is also a use case of is it "fit for purpose",  which "valid until" covers better than support.  Similarly people could use data after a supported date, as you point out,  but should they consider it "valid" - that's an open question, it's a higher risk.   Goal of this field is so that when it gets asserted, its data that can be used in policy checkers.   Note:  it is optional for 2.3, and if people don't want to indicate this, they don't have to and then it's like the status quo.  

Looking forward to 3.0 the AI profile will likely extend this concept to specific files - where training data has explicit expiry dates to force retraining.    Valid until will extend to cover this case a bit better.